components/family-studies/migrations/src/main/java/org/phenotips/studies/family/migrations/R71498PhenoTips2155DataMigration.java

```diff
@@ -710,101 +1718,11 @@ private void setAllFamilyRefs(Set<String> familyMembers, XWikiDocument familyXDo
             BaseObject familyObject = familyXDocument.getXObject(Family.CLASS_REFERENCE, true, this.context);
             familyObject.setStringListValue("members", membersRefsList);
 
-            String probandLastName = probandData.getString(LASTNAME);
-            if (!StringUtils.isEmpty(probandLastName)) {
-                familyObject.set("external_id", probandLastName, this.context);
+            if (!StringUtils.isBlank(suggestedExternalId)) {
+                familyObject.set(FAMILY_DOCUMENT_EXTERNALID_FIELD, suggestedExternalId, this.context);
```

Comment:
@marta- what do you think about this (setting family's external IDs for migrated families)?